ush/wave_grib2_sbs.sh

```diff
@@ -232,16 +255,14 @@ if [[ ! -s "${com_dir}/${outfile}.idx" ]]; then
 
   if [[ "${SENDDBN}" = 'YES' ]] && [[ ${outfile} != *global.0p50* ]]; then
     set +x
-    echo "   Alerting GRIB file as ${com_dir}/${outfile}"
-    echo "   Alerting GRIB index file as ${com_dir}/${outfile}.idx"
+    echo "   Alerting GRIB file as ${outfile}"
```

Comment:
Should the com_dir be removed from this?

Reply:
this is just an echo. its fine to be either. the crucial line is the `dbn_alert` that alerts the correct path of this file from `com_dir`.